packages/rancher-windows-gmsa/charts/questions.yaml

```diff
@@ -0,0 +1,58 @@
+questions:
+  - variable: credential.enabled
+    default: true
+    description: Whether to create a GMSA Credential Spec when installing GMSA Webhook
+    label: Whether to create a GMSA Credential Spec

```

Comment:
Whoops sorry. I missed this in my review 😅 

This should be `Whether to create a GMSA Credential`, not creating the spec right?